contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -480,6 +480,20 @@ AND Ext2.orig_username != 'db_owner';
 
 GRANT SELECT ON sys.database_role_members TO PUBLIC;
 
+--SERVER_ROLE_MEMBER
+CREATE OR REPLACE VIEW sys.server_role_members AS
+SELECT
+CAST(Authmbr.roleid AS INT) AS role_principal_id,
+CAST(Authmbr.member AS INT) AS member_principal_id
+FROM pg_catalog.pg_auth_members AS Authmbr
+INNER JOIN pg_catalog.pg_roles AS Auth1 ON Auth1.oid = Authmbr.roleid
+INNER JOIN pg_catalog.pg_roles AS Auth2 ON Auth2.oid = Authmbr.member
+INNER JOIN sys.babelfish_authid_login_ext AS Ext1 ON Auth1.rolname = Ext1.rolname
+INNER JOIN sys.babelfish_authid_login_ext AS Ext2 ON Auth2.rolname = Ext2.rolname
+WHERE Ext1.type = 'R';
```

Comment:
What about the permissions? Should all server membership should be visible by default? Please refer [doc](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/system-catalog-views/sys-server-role-members-transact-sql.md#permissions)

Reply:
In Babelfish, we only allow one server role `sysadmin` which is marked as "R" in the catalog.
In SQL Server,  the default server role `sysadmin`  is visible to everyone. There is no behavioral difference.
We don't support creating a new server role other that 'sysadmin' in Babelfish, do we?

Reply:
Yes, In future when we support it at that time we may miss it. It will be better to add a condition to check that if it is fixed server role or showing membership's for current server principal.

Reply:
Agreed.
Currently, we hardcode `is_fixed_role` to 0 always. 
I have suggested a change to Anju on how to update the `is_fixed_role` column in `babelfish_authid_login_ext` catalog which will be helpful for https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1948 as well.

Reply:
We would need to update `sys.server_role_members` view once we support `create server role` syntax in Babelfish as tracked by https://jira.rds.a2z.com/browse/BABEL-113